src/components/Player/MobileGridControls.js

```diff
@@ -492,7 +493,8 @@ export default class MobileGridControls extends GridControls {
         touchEvents: 'none',
         position: 'absolute',
       },
-      autoComplete: 'nope',
+      autoComplete: 'none',
```

Comment:
lol, nope was actually intentional (from a comment in https://gist.github.com/niksumeiko/360164708c3b326bd1c8).
but seems like it doesn't work anyway.

Reply:
Actually, autoComplete: 'none' is not a real thing either... There's [autoCapitalize: 'none' ](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autocapitalize) but no autoComplete: 'none'. Someone should change that to 'off'.